accounts/abi/bind/backends/simulated.go

```diff
@@ -681,7 +681,7 @@ func (b *SimulatedBackend) callContract(ctx context.Context, call ethereum.CallM
 
 	// Set infinite balance to the fake caller account.
 	from := stateDB.GetOrNewStateObject(call.From)
-	from.SetBalance(math.MaxBig256)
+	from.SetBalance(math.MaxBig256, 0x0)
```

Comment:
I swear I had responded on this 😮 since my response has vanished:

I don't know. I feel like it will be too easy to "forget" adding a reason if there's a method for it, specially if it's the default. But I agree the `0x0` were not so appealing so changed all of them to `Unspecified`. They were mostly in tests. The only production code was in `eth_call` state override and simulated backend.